src/Wallabag/CoreBundle/Form/Type/EditEntryType.php

```diff
@@ -30,6 +30,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => 'entry.edit.origin_url_label',
                 'default_protocol' => null,
             ])
+            ->add('content', TextareaType::class, [
```

Comment:
You should add `use Symfony\Component\Form\Extension\Core\Type\TextareaType` at the top of that file otherwise the page will display an error

Reply:
Oh shit, forgot! Thanks! Added.

Reply:
And don't forget to run `php bin/php-cs-fixer fix` to resolve coding standard issue